### PR TITLE
Add manifests to build and deploy text generation web UI

### LINKED ISSUE #27: Deploy Language Model(s) to our cluster exposing a completion API
This should be an enabler for #25 and #26, but also will help to explore open language models and how to serve them.

Goal: to deploy one or more of the publicly available Language Models to our cluster, so that they can be accessed via an API hosted by ourselves.

A comprehensive list of such models can be found in Hugging Face's Transformers' list of models, e.g. https://huggingface.co/docs/transformers/model_doc/bloom (see the list of *text models*). A summary / classification is also available here: https://huggingface.co/docs/transformers/model_summary#summary-of-the-models

### PR DESCRIPTION
**Pull request by codificat**
Here is an OpenShift manifest to build and deploy [Text generation web UI](https://github.com/oobabooga/text-generation-webui/).

It is a Gradio application that presents a web UI and also an API endpoint for inference.

Fixes #27 

I believe the previously existing manifests were not in use or relevant here, so I am also removing them.

### CODE REVIEW

File: manifests/README.md [hunk 1]
```diff
@@ -1,52 +1,19 @@
...
-      echo 'export PATH="${GOPATH//://bin:}/bin:$PATH"' >> ~/.bashrc
-      source  ~/.bashrc
-      ```
+### Requirements
 
-   3. Install `sops` from your distribution repository if possible or use [sops GitHub release binaries](https://github.com/mozilla/sops#stable-release)
+**IMPORTANT** about model persistence: the manifest does **NOT** include a `PersistentVolumeClaim` definition. However, the included `Deployment` assumes that a PVC called `llms` exists, and it is expected to be populated with the models to serve, one per directory. The `Deployment` expects to start up with `bloom-1b7`.
 
-   4. Import AICoE-SRE's public key [EFDB9AFBD18936D9AB6B2EECBD2C73FF891FBC7E](https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xefdb9afbd18936d9ab6b2eecbd2c73ff891fbc7e):
+This means that in order to use this, a PVC named `llms` **must be created** first with the appropriate content.
```
Comment (oindrillac) [inline manifests/README.md:17]: @codificat do you have the definition for the pvc? did you upload the bloom-1b7 model checkpoint into it?

Comment (codificat) [reply manifests/README.md]: @oindrillac I have now added the PVC definition to the YAML manifest and expanded the README with instructions on how to populate it

Comment (oindrillac) [reply manifests/README.md]: perfect, thanks @codificat!

**Review by oindrillac — APPROVED** [review]

**Review by Shreyanand — APPROVED** [review]
Great work on adapting the project and deploying it on our cluster! 💯

**Pull request merged**